reactor-netty-http/src/main/java/reactor/netty/http/HttpOperations.java

```diff
@@ -423,7 +424,7 @@ else if (!SCHEME_PATTERN.matcher(tempUri).matches()) {
 						ByteBuf bb = ((ByteBufHolder) msg).content();
 						ctx.fireChannelRead(bb);
 						if (msg instanceof LastHttpContent) {
-							ctx.fireChannelRead(LastHttpContent.EMPTY_LAST_CONTENT);
+							ctx.fireChannelRead(new EmptyLastHttpContent(ctx.bufferAllocator()));
```

Comment:
Is it possible to cache the EmptyLastHttpContent in some static variables and reuse it instead of allocating one each time ?